src/Serilog.Sinks.Console/Sinks/SystemConsole/Output/TimestampTokenRenderer.cs

```diff
@@ -56,5 +55,35 @@ public override void Render(LogEvent logEvent, TextWriter output)
                 }
             }
         }
+
+        readonly struct DateTimeOffsetValue
+        {
+            public DateTimeOffsetValue(DateTimeOffset value)
+            {
+                Value = value;
+            }
+
+            public DateTimeOffset Value { get; }
+
+            public void Render(TextWriter output, string? format = null, IFormatProvider? formatProvider = null)
+            {
+                var custom = (ICustomFormatter?)formatProvider?.GetFormat(typeof(ICustomFormatter));
+                if (custom != null)
+                {
+                    output.Write(custom.Format(format, Value, formatProvider));
+                    return;
+                }
+
+#if NET5_0_OR_GREATER
```

Comment:
Could we follow the Serilog convention of declaring feature flag use in the CSPROJ (i.e. `FEATURE_SPAN`) for this, here: https://github.com/serilog/serilog-sinks-console/blob/dev/src/Serilog.Sinks.Console/Serilog.Sinks.Console.csproj#L26 ?

Reply:
done